src/lib/components/forms/suggestion-box/terra-suggestion-box.component.ts

```diff
@@ -21,6 +21,7 @@ import { Language } from 'angular-l10n';
 
 const MAX_LASTLY_USED_ENTRIES: number = 5;
 
+/** @deprecated since v5. Please use mat-autocomplete of type color instead. */
```

Comment:
```suggestion
/** @deprecated since v5. Please use mat-autocomplete instead. */
```
Und ggf. auch hier nen Link dazu?